docs/topics/Teams.md

```diff
@@ -41,7 +41,7 @@ If you have an existing app that you want to transfer to a Team, you can do that
 
 ## What Next
 
-What next? Go make awesome stuff! Whether you're a Game Developer, Mad Bot Scientist, or OAuth2 Enthusiast, you can now work together with other like-minded Discordians to bring your creations to life.
+What next? Go, make awesome stuff! Whether you're a Game Developer, Mad Bot Scientist, or OAuth2 Enthusiast, you can now work together with other like-minded Discordians to bring your creations to life.
```

Comment:
this is incorrect

Reply:
I think this is correct, as `go` and `make` are two separate verbs.